pkg/validate/security_context.go

```diff
@@ -46,7 +49,7 @@ var _ = framework.KubeDescribe("Security Context", func() {
 		ic = f.CRIClient.CRIImageClient
 	})
 
-	Context("runtime should support podSandbox with security context", func() {
+	Context("runtime should support NamespaceOption", func() {
```

Comment:
why renamed this to namespace options? I think security context is more precise here.